packages/orbit-components/src/InputGroup/index.js

```diff
@@ -282,7 +282,8 @@ const InputGroup = ({
         {React.Children.toArray(children).map((item, key) => {
```

Comment:
With `React.Children.map` you don't have to worry about setting `key`! 🥳 
```suggestion
        {React.Children.map(children, item => {
```